### PR TITLE
Style admin portal gallery and submission details

### PR DESCRIPTION
**Pull request by zhangjuliet**
Are the filters at the top of Gallery Submissions redundant since the MUI DataGrid table already has filtering options?

---

Gallery screen before:
Gallery Before

Gallery screen after:
Gallery After

Submission Details screen before:
Submission Details Before

Submission Details screen after:
Submission Details After

### CODE REVIEW

File: website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx [hunk 1]
```diff
@@ -121,103 +130,184 @@ function SubmissionDetailsPage() {
...
+              {media.ContentType?.startsWith('image/') && (
+              <img
+                style={{
+                  width: '100%', borderRadius: 5, marginBottom: 20, border: '1px solid #4C4C9B',
+                }}
+                src={media.MediaURL}
+                alt={media.ContentType}
+              />
+              )}
+              {media.ContentType?.startsWith('video/') && (
```
Comment (he-is-james) [inline website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx:278]: Once you approve the gallery media render, you can decide if you want to go with your method or mine

File: website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx [hunk 2]
```diff
@@ -121,103 +130,184 @@ function SubmissionDetailsPage() {
...
-          </Typography>
+          </p>
           {(edit) ? (
             <TextField
               value={title}
               onChange={(e) => setTitle(e.target.value)}
+              inputProps={{
+                style: {
+                  color: '#53595C',
+                  border: '0.75px solid #8A9195',
```
Comment (he-is-james) [inline website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx:192]: Not sure why this is happening in my version but let me know
[](url)
Screenshot 2023-07-20 at 11 50 20 PM

**Review by he-is-james — CHANGES_REQUESTED** [review]
Overall looks really good! I like how you implemented filtering and it looks really clean.

Minor issue with how the text boxes are being rendered so double check with different zooms or window sizes maybe?

File: website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx (hunk 2 above)
Comment (zhangjuliet) [reply website/src/Pages/GalleryPage/SubmissionDetailsPage.jsx:192]: @he-is-james I'm unable to recreate this issue on my end. Is it still happening?

**Review by he-is-james — APPROVED** [review]

**Pull request merged**